src/components/Sidebar/SidebarTree/TreeInfos.tsx

```diff
@@ -309,16 +334,12 @@ const TreeInfos: FC<{
               }
               type='secondary'
             >
-              {adoptedByLoggedInUser &&
-                !isBeingUnadopted &&
-                'Adoption aufheben'}
+              {adoptedByLoggedInUser && !isBeingUnadopted && stopAdoption}
               {adoptedByLoggedInUser &&
                 isBeingUnadopted &&
-                'Adoption wird aufgehoben'}
-              {!adoptedByLoggedInUser && !isBeingAdopted && 'Baum adoptieren'}
-              {!adoptedByLoggedInUser &&
-                isBeingAdopted &&
-                'Baum wird adoptiert'}
+                stopAdoptionProgress}
+              {!adoptedByLoggedInUser && !isBeingAdopted && adopt}
+              {!adoptedByLoggedInUser && isBeingAdopted && adoptProgress}
```

Comment:
I'd suggest to wrap the localized content in a `<>{...}</>` to make it easier to understand that the last variable in the condition is actually the string content to be used and not a boolean. (for all three lines above)

Reply:
good point